src/sonic-yang-models/yang-events/sonic-events-pmon.yang

```diff
@@ -0,0 +1,42 @@
+module sonic-events-pmon {
+    namespace "http://github.com/sonic-net/sonic-events-pmon";
+    yang-version 1.1;
+
+    import openconfig-alarm-types {
+        prefix oc-alarm-types;
+    }
+
+    import sonic-events-common {
+        prefix evtcmn;
+    }
+    
+    revision 2022-12-01 {
+        description "pmon alert events.";
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC pmon events";
+
+
+    container pmon-exited {
```

Comment:
Top-level container

Reply:
YANG review is moved to different [PR #11404 ](https://github.com/Azure/sonic-buildimage/pull/11404)